sihl-email/src/sihl_email.ml

```diff
@@ -358,18 +351,20 @@ module Queued
     then (
       Logs.debug (fun m -> m "Skipping queue for email sending");
       Email.send email)
-    else QueueService.dispatch Job.job email
+    else Job.dispatch email
   ;;
 
   let bulk_send emails =
-    (* TODO [jerben] Implement queue API for multiple jobs so we don't have to
-       use transactions here *)
-    let rec loop emails =
-      match emails with
-      | email :: emails -> Lwt.bind (send email) (fun () -> loop emails)
-      | [] -> Lwt.return ()
-    in
-    loop emails
+    if not (Sihl.Configuration.is_production ())
+    then (
+      Logs.debug (fun m -> m "Skipping queue for email sending");
```

Comment:
Please document this somewhere

Reply:
I just checked and we don't yet have documentation for `sihl-email`. There is an issue for that though.

Reply:
Ok, thanks